My-Portfolio/script.js

```diff
@@ -230,3 +230,25 @@ previewWindow.forEach((close) => {
     background.style.filter = 'none';
   };
 });
+
+const email = document.getElementById('email');
+const form = document.getElementById('formContainer');
+
+function validateEmail() {
+  const emailRegExp = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+  const isNotValid = !emailRegExp.test(email.value);
+
+  if (isNotValid) {
+    form.setCustomValidity(
+      'You have UPPERCASE in the email you typed, Please use lowercase!',
+    );
+  } else email.setCustomValidity('');
+}
+
+email.addEventListener('input', () => {
+  validateEmail();
+});
+
+form.addEventListener('submit', () => {
+  validateEmail();
+});
```

Comment:
I can see you made effort in adding validations to your email field👍. But I think you might have some bugs in your validations. currently, when I enter an email with some upper-case characters, it proceeds to submission without giving me any error. don't forget it is a[ requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md) that an error should be displayed when the content of our email field has uppercase characters😉😉 
![Screenshot from 2023-02-02 16-08-00](https://user-images.githubusercontent.com/64643995/216363991-9bd5c6ea-2ba4-47e2-87fc-ae5c17681db0.png)
I think it is because you have attached the event listener to the email field, I think the event listener should be added to the `get in touch` button instead😉

I suggest you kindly go over the requirements and make the necessary adjustments🤗🤗

Reply:
corrected :D